source/docs/advanced/theme/analytics.md

```diff
@@ -0,0 +1,237 @@
+---
+id: analytics
+title: Analytics
+---
+
+Most e-commerce website need full blown analytics to better understand their users and adapt their shops to their customers needs. But it can often be tedious to manage when you have many trackings to manage.
```

Comment:
```suggestion
Most e-commerce website need advanced analytics to better understand their users and adapt their shops to their customers needs. But it can often be tedious to maintain when you have many trackings to manage.
```